pymech/readma2.py

```diff
@@ -0,0 +1,76 @@
+def readma2(fname,ldim):
+    """A function for reading binary map files (*.ma2) for nek5000.
+
+    The map file comtains, for each element in the mesh, the id of the MPI rank that owns it 
+    followed by the ids of the vertices of the element in the global address space.
+
+    The partitioning is determined by generating the undirected graph formed by the mesh, then 
+    repeatedly computing and deviding the graph using the Fiedler vector of the graph Laplacian. 
+
+    Parameters
+    ----------
+    fname : str
+            file name
+    ldim  : int
+            dimension of the mesh
+    """
```

Comment:
Describe the return values (type) and what it contains in the docstring here (just like you did for Parameters).

Reply:
Good point, I have added the information about the return values.